index.html

```diff
@@ -41,6 +45,177 @@ <h1 class="headline-heading">I'm Usman <br> Glad to see you!</h1>
             </ul>
         </section>
     </section>
+    <!-- Headline Section Ends Here -->
+
+    <!-- Works Section Starts Here -->
+    <!-- Card 1 starts -->
+
+    <section class="works-card">
+        <img src="./assets/works-1.png" alt="Recent Work">
+        <h1 class="works-card-heading-large">Tonic</h1>
+        <section class="works-card-client">
+            <p class="paragraph">CANOPY</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-role paragraph">Back End Dev</p>
+            <i class="fa-solid fa-circle works-card-client-counter"></i>
+            <p class="works-card-client-year paragraph">2015</p>
+        </section>
+        <p class="paragraph works-card-paragraph">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <section class="works-card-tags">
+            <p class="works-card-tag">html</p>
+            <p class="works-card-tag">css</p>
+            <p class="works-card-tag">javascript</p>
+        </section>
```

Comment:
The design is fabulous, but it's important to use the `ul` and `li` here because you're listing the technologies in here.